docs/management/index-lifecycle-policies/example-index-lifecycle-policy.asciidoc

```diff
@@ -1,23 +1,167 @@
 [role="xpack"]
+
 [[example-using-index-lifecycle-policy]]
-=== Example of using an index lifecycle policy
+=== Tutorial:  Use ILM to manage Filebeat time-based indices
+
+With index lifecycle management (ILM), you can create policies to define actions
+that are automatically performed against indices as they age.  In this tutorial,
+you will use {kib}’s *Index Lifecycle Policies* to modify and create ILM policies.
+
+
+[float]
+==== Scenario
```

Comment:
We should use `[discrete]` instead of `[float]` when possible. I'd also add an explicit anchor here.

```suggestion
[discrete]
[[example-using-index-lifecycle-policy-scenario]]
==== Scenario
```